src/mame/misc/falgas_m89.cpp

```diff
@@ -151,16 +201,52 @@ void falgasm89_state::cbully(machine_config &config)
 	psg.port_b_read_callback().set(FUNC(falgasm89_state::psg_pb_r));
 }
 
+void falgasm89_state::falgasm89(machine_config &config)
+{
+	falgasm89_simple(config);
+
+	I8255(config, "i8255"); // NEC D71055C
+}
```

Comment:
What actually uses this configuration?  As far as I can tell, it isn’t used by anything right now.  Does the declaration need to be marked as unused?  What’s the 8255 actually used for when it’s present?  More I/O?

Reply:
Oh my bad, the 8255 is used with the video version.  Are there actually games that use the 8255 without the video hardware, or does the 8255 only need to be populated to communicate with the video hardware?

Reply:
It's hard to say for sure... There are about seven machines using M89 hardware, and just one with video. I'd say that yes, there are other machines without video that use the 8255 (more buttons, sensors, etc.).